package.json

```diff
@@ -23,8 +23,10 @@
     "format": "yarn lint --fix"
   },
   "dependencies": {
+    "ajv": "^7.1.0",
```

Comment:
change to `>= 7.1.0 < 8`

Reply:
No need. That's already what ^7.1.0 means.